### PR TITLE
docs: RENOVATE_CONFIG_FILE environment variable

### PR DESCRIPTION
**Pull request by JuliusHenke**
## Changes:

Documents the `RENOVATE_CONFIG_FILE` environment variable.

## Context:

Since renovate v30.0.0 only the `config.js` file is supported by default. I could not find the documentation suggesting to set the `RENOVATE_CONFIG_FILE` variable to support the other global config file names.
https://github.com/renovatebot/renovate/releases/tag/30.0.0

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/getting-started/running.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ WhiteSource Renovate On-Premises and WhiteSource Remediate both run as long-live
 ### Global config
 
 Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file (`config.js`, `config.json`, `config.json5`, `config.yaml` or `config.yml`), environment variables, or CLI parameters.
```
Comment (rarkins) [inline docs/usage/getting-started/running.md:95]: I think that this line above is perhaps misleading. By default we only support `config.js`, but we can now support any file of any name with `.json`, `.json5`, `.yaml` or `.yml` file type

Comment (JuliusHenke) [reply docs/usage/getting-started/running.md]: Yes, I agree. I updated the paragraph. I believe the any `*.js` named file is also supported with the `RENOVATE_CONFIG_FILE` variable set?

Comment (viceice) [reply docs/usage/getting-started/running.md]: Yes

File: docs/usage/getting-started/running.md [hunk 2]
```diff
@@ -92,8 +92,7 @@ WhiteSource Renovate On-Premises and WhiteSource Remediate both run as long-live
 
 ### Global config
 
-Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file (`config.js`, `config.json`, `config.json5`, `config.yaml` or `config.yml`), environment variables, or CLI parameters.
-If you are using a global config file other than `config.js`, you need to set the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=/usr/src/app/config.json`).
+Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`. Any other (`*.js`, `*.json`, `*.json5`, `*.yaml` or `*.yml`) file is supported, when you reference it with the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=config.yaml`).
```
Comment (rarkins) [inline docs/usage/getting-started/running.md]:
```suggestion
Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`.
Any other (`*.js`, `*.json`, `*.json5`, `*.yaml` or `*.yml`) file is supported, when you reference it with the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=config.yaml`).
```

File: docs/usage/getting-started/running.md [hunk 3]
```diff
@@ -92,7 +92,8 @@ WhiteSource Renovate On-Premises and WhiteSource Remediate both run as long-live
 
 ### Global config
 
-Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`. Other file types and names (`*.js`, `*.json`, `*.json5`, `*.yaml`, `*.yml`) are supported, when you set the `RENOVATE_CONFIG_FILE` environment variable (e.g. `RENOVATE_CONFIG_FILE=config.yaml`).
+Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters. Renovate autodetects a global config file named `config.js`.
```
Comment (rarkins) [inline docs/usage/getting-started/running.md:95]:
```suggestion
Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters.
By default Renovate checks if a file named `config.js` if present.
```

**Review by rarkins — CHANGES_REQUESTED** [review]

File: docs/usage/getting-started/running.md (hunk 3 above)
Comment (JuliusHenke) [reply docs/usage/getting-started/running.md:95]: In my opinion this misses the point of documenting the RENOVATE_CONFIG_FILE variable.

Comment (rarkins) [reply docs/usage/getting-started/running.md:95]: I'm confused:
1. My suggestion is to add a new line, according to our markdown convention
2. There's still the sentence below

Comment (JuliusHenke) [reply docs/usage/getting-started/running.md]: Ahh got you. For some reason I was not able to commit GitHub review suggestions on the spot 😄

File: docs/usage/getting-started/running.md [hunk 4]
```diff
@@ -92,7 +92,9 @@ WhiteSource Renovate On-Premises and WhiteSource Remediate both run as long-live
 
 ### Global config
 
-Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file (`config.js`, `config.json`, `config.json5`, `config.yaml` or `config.yml`), environment variables, or CLI parameters.
+Renovate's server-side/admin config is referred to as its "global" config, and can be specified using either a config file, environment variables, or CLI parameters.
+By default Renovate checks if a file named `config.js` if present.
```
Comment (rarkins) [inline docs/usage/getting-started/running.md:96]:
```suggestion
By default Renovate checks if a file named `config.js` is present.
```

**Review by rarkins — APPROVED** [review]

**Pull request merged**